packages/next-swc/crates/core/tests/errors/server-actions/server-graph/3/output.js

```diff
@@ -1,9 +1,11 @@
-/* __next_internal_action_entry_do_not_use__ x */ export const x = 1;
-import ensureServerEntryExports from "private-next-rsc-action-proxy";
+/* __next_internal_action_entry_do_not_use__ x */ import __create_action_proxy__ from "private-next-rsc-action-proxy";
+export const x = 1;
+import ensureServerEntryExports from "private-next-rsc-action-validate";
 ensureServerEntryExports([
     x
 ]);
 x.$$typeof = Symbol.for("react.server.reference");
 x.$$id = "b78c261f135a7a852508c2920bd7228020ff4bd7";
 x.$$bound = null;
 x.$$with_bound = false;
+__create_action_proxy__(x);
```

Comment:
It might be beneficial to just pass the id and closure bound arguments in here as arguments to `__create_action_proxy__`. Since then the hidden class only needs to be messed with once. And `x.$$typeof = Symbol.for("react.server.reference");` can just be in one place instead of duplicated everywhere since it's always repeated.